ynr/apps/candidates/management/commands/candidates_import_statements_of_persons_nominated.py

```diff
@@ -161,18 +164,28 @@ def handle(self, *args, **options):  # noqa
             with open(downloaded_filename, "rb") as f:
                 storage_filename = storage.save(filename, f)
 
-            try:
-                OfficialDocument.objects.create(
-                    document_type=OfficialDocument.NOMINATION_PAPER,
-                    uploaded_file=storage_filename,
-                    ballot=ballot,
-                    source_url=document_url,
-                )
-            except PandocConversionError:
-                self.stderr.write(
-                    f"Error attempting to convert {document_url} to a PDF, skipping this row"
-                )
-                continue
+            # create with the original file
+            official_document = OfficialDocument.objects.create(
+                document_type=OfficialDocument.NOMINATION_PAPER,
+                uploaded_file=storage_filename,
+                ballot=ballot,
+                source_url=document_url,
+            )
+
+            if extension != ".pdf":
+                try:
+                    new_file = convert_sopn_to_pdf(
+                        official_document.uploaded_file.file
+                    )
+                    official_document.uploaded_file.save(
+                        storage_filename.replace(extension, ".pdf"),
+                        new_file.file,
+                    )
+                except PandocConversionError:
+                    self.stderr.write(
+                        f"Error attempting to convert {document_url} to a PDF, skipping this row"
+                    )
```

Comment:
should this be `{source_url}`?

Reply:
`document_url` is a var set earlier up in the code, which then gets used to set the `source_url`. So they are the same thing basically